my-dsl.md

```diff
@@ -2,37 +2,46 @@
 _What is the name of the language? Link the name to its webpage 
 (if appropriate)._
 
+The name of the language is CHEM. It is fairly old, so it does not have a webpage. However, a paper that talks about it can be found at this link: http://dl.acm.org/citation.cfm?id=315691
```

Comment:
The [paper](http://dl.acm.org/citation.cfm?id=315691) you reference is _the_ classic paper in DSLs. We don't read it in our class because lots of knowledge has been added since the paper was published. But feel free to use it as an additional resource in this week's assignment on language design!